### ISSUE TITLE
Editing a Test Case in a Suite Should Not Update in a Run

### ISSUE THREAD

**Issue by lgranneman** [ToucanTesting/ToucanTesting#6]
**Describe the bug**
Currently if a test case is edited in a suite, it also updates the test case in the run.  
On the flip side, editing a test case in a run should update in the suite and that is the current behavior.

**Expected behavior**
Updating a test case in a suite should not change a run.

**Repro Steps**
1. Open a run
2. Open a module
3. Click on a test case to open it
4. Navigate to the Test Actions tab
5. Type "test" in the add an action field
6. Click Add
7. Navigate to the Suites
8. Open the same module
9. Open the same test
10. Select the Test Actions tab
11. Notice that "test" is listed

[Toucan - Editing Test Case In Suite Should Not Update Run.zip](https://github.com/ToucanTesting/ToucanTesting/files/2001427/Toucan.-.Editing.Test.Case.In.Suite.Should.Not.Update.Run.zip)

**Comment by djbreen7** [comment]
This actually isn't a bug. I intentionally didn't go this deep into test cases. Lean Testing worked the same way in this case.

There's too much complexity to efficiently keep steps/conditions/results updated as this would mean saving every edit as a new database entry. For example, if you have a typo in a test case, fix the typo, then decide you want title case, 3 entries get added to the database and previous test runs would have to track which step was active at the time of the test run creation.

Basically, the complexity is exponential times 3 (There are over 1000 test cases, each with a guessed average of 20 steps, 2 conditions, and 1 expected result). So at the least, we'd be trying to sort data between 23,000 rows and each test case is potentially another 23 rows. Rewriting/editing the steps just once would make it 43, and so on.
